README.md

```diff
@@ -107,6 +110,28 @@ const client = new Client({
 });
 ```
 
+#### DPoP authorization
+
+The client supports Oauth2+DPoP authorization ONLY FOR INTERNAL USE in service-to-service. Required methods can be configured during the initialization as 'authStrategy.' You must provide your implementation of the `getAccessToken` and `getDpopHeader` methods.
```

Comment:
```suggestion
The client supports Oauth2+DPoP authorization ONLY FOR INTERNAL USE in service-to-service. Required methods can be configured during the initialization as 'authStrategy.' You must provide your implementation of the `getAccessToken` and `getDpopHeader` methods.
```

That's an issue, as the current implementation is also used outside and not internally only. So if the current implementation is only meant for LP Services, we should make sure the method reflects that. As `getDpopHeader` appears very not internal

Reply:
changed